src/test/Spec.hs

```diff
@@ -0,0 +1,120 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ViewPatterns #-}
+
+import Test.Tasty
+import Test.Tasty.HUnit as HU
+
+import Echidna.Campaign (Campaign(..), tests, campaign, TestState(..))
+import Echidna.Config (EConfig, defaultConfig, sConf, parseConfig)
+import Echidna.Solidity (loadTesting, quiet)
+import Echidna.Test (SolTest)
+import Echidna.Transaction (Tx, call)
+
+import Control.Lens ((^.), (&), (.~), view)
+import Control.Monad.Reader (runReaderT)
+import Data.Maybe (fromJust)
+import Data.Text (Text)
+import EVM.ABI (AbiValue(..))
+
+main :: IO ()
+main = defaultMain spec
+
+spec :: TestTree
+spec = testGroup "Echidna" [solidityTests]
+
+solidityTests :: TestTree
+solidityTests = testGroup "Solidity-HUnit"
+  [ testCase "Always True" $ testContract' c5 $
+      \c -> do
+        let tr = findtest' c "echidna_true"
+        assertBool "echidna_true somehow did not pass" $ passed tr
+  , testCase "Simple Flags Example" $ testContract' c2 $
+      \c -> do
+        let t1 = findtest' c "echidna_alwaystrue"
+            t2 = findtest' c "echidna_sometimesfalse"
+        assertBool "echidna_alwaystrue did not pass" $ passed t1
+        assertBool "echidna_sometimesfalse unsolved" $ solved t2
+        let sol = solve t2
+        assertBool "solution has length /= 2" $ length sol == 2
+        -- the following check will fail with significantly higher probability
```

Comment:
Rather than checking in commented code, I think we should get this out of the file and into a github issue, where it's easier to manage

Reply:
Done (@japesinator)